articles/applications/application-settings/machine-to-machine/index.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Application settings for Machine to Machine Applications
+toc: true
+---
+
+# Application Settings: Machine to Machine Application
+
+::: version-warning
+This document lists the settings for a Machine to Machine Application; if you're using a different Application type, please use the drop-down to select the appropriate doc.
+:::
+
+When creating an Auth0 Application, you'll be asked to indicate the *type* of Application you want to create. 
+
+![Window for selecting application type](/media/articles/clients/create-clients.png)
+
+If your app is a CLI, daemon, or a service running on the backend, you'll want to create a **Machine to Machine Application**.
+
+## Settings
+
+<%= include('../_settings') %>
+
+- **Application Type**: The type of application you are implementing. If your app is a CLI, daemon, or a service running on the backend, use a **Machine to Machine Application**.
```

Comment:
`If your app is a CLI, daemon, or a service running on the backend, use a **Machine to Machine Application**.`
The part before the comma is redundant (since the doc is specific to machine to machine)